docs/_i18n/en.yml

```
@@ -24,4 +24,13 @@ pages:
   private_data_exchange: Private data exchange
   reference: Reference
   tutorials: Tutorials
-  understanding_firefly: Understanding FireFly
\ No newline at end of file
+  understanding_firefly: Understanding FireFly
+  chains: Connecting to other chains
```

I'm not sure what "other" is referring to here.

```suggestion
  chains: Connecting remote blockchains
```